run_python_examples.sh

```diff
@@ -164,6 +150,12 @@ function gat() {
   uv run main.py --epochs 1 --dry-run || error "graph attention network failed"
 }
 
+function differentiable_physics() {
+  pushd differentiable_physics
+  python -m uv run mass_spring.py --mode train --epochs 5 --steps 3 || error "differentiable_physics example failed"
+  popd
```

Comment:
What's the reason to use different pattern for tests just for this example? One potential problem - `python` might not actually exist on the system as typically you get `python3` and need to create a symbolic link to it in one way or another.

```suggestion
  uv run mass_spring.py --mode train --epochs 5 --steps 3 || error "differentiable_physics example failed"
```

Reply:
Rectified it.